package.json

```diff
@@ -13,7 +13,9 @@
     "dev": "yarn run serve",
     "docs": "jsdoc -c jsdoc.conf.json",
     "lint": "vue-cli-service lint",
-    "serve": "vue-cli-service serve --mode offline",
+    "serve": "yarn run serve:jupyterhub & yarn run serve:vue",
```

Comment:
I wish NPM and Yarn supported multiple commands & command chaining. This isn't so nice as if `yarn run serve:jupyterhub` fails, then the other command continues running and you cannot stop it easily with `CTRL + C`. The alternatives are installing packages like `npm-run-all`, `npm-or-yarn`, or starting both commands separately :man_shrugging:

Reply:
Using https://github.com/kimmobrunfeldt/concurrently#readme, after reading about it in several posts. Simpler than `npm-run-all` or `yarn-run-all`, but also doesn't appear to lock us with `npm` nor `yarn`. Also liked the option to have a prefix with some color to identify where the log is coming from (similar to `docker-compose`).